libevm/VMFactory.cpp

```diff
@@ -133,7 +133,7 @@ po::options_description vmProgramOptions(unsigned _lineLength)
     add("vm",
         po::value<VMKind>()
             ->value_name("<name>")
-            ->default_value(VMKind::Interpreter, "interpreter")
+            ->default_value(VMKind::Legacy, "legacy")
```

Comment:
What is the legacy vm vs. the interpreter?

Reply:
The same implementation but different interfaces. I keep the "legacy" one always have a working one and keep the vm tracing support.